src/pages/components/ConfigDialog.qml

```diff
@@ -1,6 +1,6 @@
 /****************************************************************************
 **
-** Copyright (C) 2013
+** Copyright (C) 2014
```

Comment:
Correct form used in oss licenses is 2014-2015 (year of first realease - year of last release)